elements/templates/a-ring.html

```diff
@@ -0,0 +1,32 @@
+<template is="a-template"
+          element="a-ring"
+          inner-radius="1"
```

Comment:
you don't want to call this `radius-inner` and `radius-outer` for consistency's sake?